### ISSUE TITLE
what are the tamperprotectionsources?

### ISSUE THREAD

**Issue by schrx1964** [MicrosoftDocs/microsoft-365-docs#9219]
I see with the powershell command get-mpcomputerstatus tamperprotectionsource = signatures, sometimes ATP and sometimes E5 transitioning.
Is there any documentation about the the different sources?
Other point is that my AVD devices getting a greyed out option on the tamperprotection with tamperprotection off once it is onboarded in defender for endpoint. When i try to turn it on through MEM i get not applicable on that antivirus policy.
How do i get tamperprotection to On state without turning it on in the advanced settings in the security portal

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f31e7b39-b7c3-188a-0b2a-02ff10a5be6d
* Version Independent ID: 44940a13-0a2d-e281-560b-cb4d8a5d1ccf
* Content: [Protect security settings with tamper protection](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection?view=o365-worldwide#turn-tamper-protection-on-or-off-in-the-microsoft-365-defender-portal)
* Content Source: [microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md)
* Product: **m365-security**
* Technology: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

**Comment by franklesniak** [comment]
A value of 3 = "E3 transition", whatever that means :)

**Comment by denisebmsft** [comment]
@yogkumgit I have opened a work item (6329673) for this issue. Will update this issue when I know more.

**Comment by denisebmsft** [comment]
Hello @yogkumgit. We've been waiting on information from the PM. We're on hold until we hear back from him.

**Comment by denisebmsft** [comment]
Hello @schrx1964 and thank you for posting your feedback here. I finally have an update! The sources for configuring tamper protection are as follows:

- The Microsoft 365 Defender portal (turn tamper protection on or off, tenant wide) - See https://learn.microsoft.com/microsoft-365/security/defender-endpoint/manage-tamper-protection-microsoft-365-defender
- Intune (turn tamper protection on or off, tenant wide) - See https://learn.microsoft/microsoft-365/security/defender-endpoint/manage-tamper-protection-microsoft-endpoint-manager
- Configuration Manager (with tenant attach, you can configure tamper protection for some or all devices by using the Windows Security experience profile) - See https://learn.microsoft.com/microsoft-365/security/defender-endpoint/manage-tamper-protection-configuration-manager 
- Windows Security app (for an individual device used at home or that is not centrally managed by a security team) - See https://learn.microsoft.com/microsoft-365/security/defender-endpoint/manage-tamper-protection-individual-device

We are adding this information to our FAQ for tamper protection: https://learn.microsoft.com/microsoft-365/security/defender-endpoint/faqs-tamper-protection

I hope this helps! Apologies for this taking a while to address. Please let me know if you still have questions.

**Comment by Phildo76** [comment]
The articles don't answer the original question. I'm experiencing the same issue on AVD devices. TamperProtectionSource = Signatures and IsTamperProtection = False!!!. Tamperprotection is switched on on the so should be environment wide and I'd expect TamperProtection to be on with the source as ATP. AVDs are configured via MECM without tenant attach for the Endpoint Protection Settings.
